pkg/schema/schema.go

```diff
@@ -20,8 +20,10 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"os/user"
 	"strings"
 
+	"github.com/elotl/cloud-init/config"
```

Comment:
We can create follow-up cards, but at the moment I don't see any actionable item... as it would basically mean we would have to fork and maintain our own :man_shrugging: